modules/bank.js

```diff
@@ -116,7 +118,7 @@ async function slashBankGive(interaction) {
     if ((currency == "em") && toIcarus) {
       const hoh = interaction.client.channels.cache.get(sf.channels.headsofhouse);
       const hohEmbed = u.embed()
-      .setAuthor(interaction.client.user.username, interaction.client.user.displayAvatarURL({ dynamic: true }))
+      .setAuthor({ name: interaction.client.user.username, iconURL: interaction.client.user.displayAvatarURL({ dynamic: true }) })
```

Comment:
This is one of the few places that did need updating. Keep the change you made here (or even better, move the author into the call to u.embed() as is done in most other places).